regolith/dates.py

```diff
@@ -239,3 +238,30 @@ def get_due_date(thing):
     else:
         raise RuntimeError(f'due date not a known type')
     return due_date
+
+def is_current(thing, now=None):
+    """
+    given a thing with dates, returns true if the thing is current
+    looks for begin_ and end_ daty things (date, year, month, day), or just
+    the daty things themselves. e.g., begin_date, end_month, month, and so on.
+    Parameters
+    ----------
+    thing: dict
+      the thing that we want to know whether or not it is current
+    now: datetime.date object
+      a date for now.  If it is None it uses the current date.  Default is None
+    Returns
+    -------
+    bool
+    """
+    if not now:
+        now = datetime.date.today()
+    dates = get_dates(thing)
+    current = False
+    if not dates['begin_date']:
```

Comment:
Can you try this?
```
is_current = False
try:
    if dates.get("begin_date") <= now <= dates.get("end_date", datetime.date(5000, 12, 31)):
        is_current = True
except:
    raise RunTimeError("something helpful here")
return is_current
```
This is better if you specify the actual exception that would be thrown by there being no begin date in the except statement so that other exceptions will cause a crash.